From d0efd0f0114bfa31eb881f7445037ae93a99f566 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Mon, 13 Nov 2017 14:46:10 -0800
Subject: [PATCH] PUBAPI-1458 CA tests broken by changes in error objects'
 structure Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by:
 Marsell Kukuljevic <marsell@joyent.com>

---
 test/analytics.test.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/analytics.test.js b/test/analytics.test.js
index c63a5f3..c9bd8ca 100644
--- a/test/analytics.test.js
+++ b/test/analytics.test.js
@@ -690,6 +690,8 @@ test('DeleteInstrumentation OK', function (t) {
             t.equal(res2.statusCode, 404);
 
             t.deepEqual(err2, {
+                jse_info: {},
+                jse_shortmsg: '',
                 message: 'resource not found',
                 statusCode: 404,
                 restCode: 'ResourceNotFound',
-- 
2.21.0

